installer/dev/install.h

```diff
@@ -15,6 +15,7 @@ namespace WindowsAppRuntimeInstaller {
         Quiet               = 0x0004,
         DryRun              = 0x0008,
         ForceDeployment     = 0x0010,
+        RepairPackages      = 0x0020,
```

Comment:
Does this only repair packages? I didn't see an option `--repair-packages`

SUGGESTION: Rename `Repair`